specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2025-03-01-preview/healthcare-apis.json

```diff
@@ -0,0 +1,4329 @@
+      "readOnly": true,
+      "properties": {
+        "createdBy": {
+          "type": "string",
+          "description": "The identity that created the resource."
+        },
+        "createdByType": {
+          "type": "string",
+          "description": "The type of identity that created the resource.",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "x-ms-enum": {
+            "name": "createdByType",
+            "modelAsString": true
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of resource creation (UTC)."
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "The identity that last modified the resource."
+        },
+        "lastModifiedByType": {
+          "type": "string",
+          "description": "The type of identity that last modified the resource.",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "x-ms-enum": {
+            "name": "createdByType",
+            "modelAsString": true
+          }
+        },
+        "lastModifiedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of resource last modification (UTC)"
+        }
+      }
+    },
+    "ResourcePublicNetworkAccess": {
+      "description": "Control permission for data plane traffic coming from public networks while private endpoint is enabled.",
+      "enum": [
+        "Enabled",
+        "Disabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "PublicNetworkAccess",
+        "modelAsString": true
+      },
+      "readOnly": true
+    },
+    "Encryption": {
+      "description": "Settings to encrypt a service",
+      "type": "object",
+      "properties": {
+        "customerManagedKeyEncryption": {
+          "type": "object",
+          "description": "The encryption settings for the customer-managed key",
+          "readOnly": false,
+          "properties": {
+            "keyEncryptionKeyUrl": {
+              "type": "string",
+              "description": "The URL of the key to use for encryption",
+              "readOnly": false
+            }
+          }
+        }
+      }
+    },
+    "StorageConfiguration": {
+      "description": "The configuration of connected storage",
+      "type": "object",
+      "properties": {
+        "storageResourceId": {
+          "type": "string",
+          "description": "The resource id of connected storage account.",
+          "readOnly": false
+        },
+        "fileSystemName": {
+          "type": "string",
+          "description": "The filesystem name of connected storage account.",
+          "readOnly": false
+        },
+        "storageMonitorConfiguration": {
```

Comment:
This is the new addition to the API. A new property called `storageMonitorConfiguration` and its definition below